eslint.config

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
See link for reference of how your config file should look:
https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json